.github/workflows/promote.yml

```diff
@@ -0,0 +1,116 @@
+
+name: Promote to channel
+
+env:
+  ORG: kyma-project
+  BTP_MANAGER_REPO: btp-manager
+  KYMA_REPO: kyma
+  TEMPLATE_NAME: moduletemplate-btp.yaml
+  GIT_EMAIL: <>
+  GIT_NAME: BTP Manager
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseTag:
+        description: 'Release Tag'
+        default: use latest available release 
+        required: true
+      channel:
+        description: 'Channel'
+        type: choice
+        options:
+        - alpha
+        - fast
+        - regular
+        required: true
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Use default Release Tag
+      if: inputs.releaseTag == 'use latest available release' 
+      shell: bash
+      run: |
+        latest=$(curl \
+                  -H "Accept: application/vnd.github+json" \
+                  -H "X-GitHub-Api-Version: 2022-11-28" \
+                  https://api.github.com/repos/$ORG/$BTP_MANAGER_REPO/releases/latest | jq -r '.tag_name') 
+
+        echo "TAG=${latest}" >> $GITHUB_ENV
+
+    - name: Validate Given Relase Tag
+      if: inputs.releaseTag != 'use latest available release'
+      shell: bash
+      run: |
+          tags=$(curl \
+                  -H "Accept: application/vnd.github+json" \
+                  -H "X-GitHub-Api-Version: 2022-11-28" \
+                  https://api.github.com/repos/$ORG/$BTP_MANAGER_REPO/tags | jq -r '.[] | .name')
+          
+          if  echo $tags | tr " " '\n' | grep -F -q -x ${{ inputs.releaseTag }}; then
```

Comment:
Why do we replace spaces with new lines? Aren't the tags returned already line separated?

Reply:
```
➜  github-actions tags=$(curl \ 
                   -H "Accept: application/vnd.github+json" \
                   -H "X-GitHub-Api-Version: 2022-11-28" \
                   https://api.github.com/repos/kyma-project/btp-manager/tags | jq -r '.[] | .name')
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   965  100   965    0     0   3287      0 --:--:-- --:--:-- --:--:--  3362
➜  github-actions echo $tags
0.1.0
0.0.1
```

Reply:
But nevertheless this replacement does not hurt.